src/app.ts

```diff
@@ -90,13 +101,13 @@ app.use((req: Request, res: Response, next: NextFunction) => {
 routerDispatch(app);
 
 // http-error error handler
-app.use(httpErrorHandler);
+app.use(...Object.values(errorHandler));
```

Comment:
Not a fan of using `...Object.values` here, would prefer an explicit import and reference to the handlers i.e:

```suggestion
app.use(errorHandler.httpErrorHandler);
app.use(errorHandler.csrfErrorHandler);
```

Just makes it a bit more obvious what is being used, also with the way it is now any other functions exported from `errorController.ts` would implicitly get used here, which might seem like unexpected behaviour/confusing if someone doesn't know about it.  Not too big of a deal though! just a suggestion.

Reply:
Done as suggested